fix doc "Artifact transform selection and execution" (#22995)
gradle/gradle: subprojects/docs/src/docs/userguide/dep-man/04-modeling-features/artifact_transforms.adoc

<2> All JAR files are not minified
<3> Request `minified=true` on all resolvable configurations
<4> Add the dependencies which will be transformed

> <5> Add task to execute the artifact transformation

Reviewer: "execute" here makes it sound like resolveRuntimeClasspath is doing the work of the artifact transform.

What about...
```suggestion
<5> Add task that requires the transformed artifacts
```

> <5> Create Task to execute the artifact transformation

Reviewer 2: 💄 The rest of the bullet points use "Add" as the verb, I would do the same here for consistency.
